### DISCUSSION TITLE
remote sensors

### DISCUSSION THREAD

**Discussion by dansrogers** [mKeRix/room-assistant#884] [General]
it would be great if room assistant could consume non-device sensors.  I.e. over the network via mqtt.  That would allow i.e. one high powered raspberry pi per room (or per several rooms), but carefully placed esphome or pi zero sensor platforms to be placed strategically throughout the room.

**Comment by mKeRix** [comment]
Is that mostly for integrating them into the Bluetooth room logic or also for other things? Might also slightly go in the direction of #694.

**Reply by dansrogers** [reply]
yes, exactly.  Also relevant for sensor fusion

**Reply by EdJoJob** [reply]
For me, I have a few old Android devices which have the HA Companion installed, which has a BLE monitor.  I would love if I could somehow bring the details from those devices to room-assistant as sensors in different rooms, without needing to acquire and set-up additional devices in those locations only for room-assistant to run on.
